pyphi/models/actual_causation.py

```diff
@@ -163,6 +164,14 @@ def purview(self):
         """
         return self._ria.purview
 
+    @property
+    def extended_purview(self):
+        """tuple of tuple[int]: List of purviews over which this causal link is
+        (equivalently) maximally irreducible.
```

Comment:
Put a newline after the first line of the docstring. Consider trying to make it fit on one line.